src/components/RecentSubmission.js

```diff
@@ -6,7 +6,7 @@ const RecentSubmission = (props) => {
   return (
```

Comment:
Just note there's a prop `submission` that you're using here which is marked as required in propTypes, but it's not being used.  Either remove it, or use the prop to get rid of the warning.